R/recipe.R

```diff
@@ -12,7 +12,10 @@ recipe <- function(x, ...) {
 #' @rdname recipe
 #' @export
 recipe.default <- function(x, ...) {
-  rlang::abort("`x` should be a data frame, matrix, or tibble")
+  cli::cli_abort(c(
+    x = "{.arg x} should be a data frame, matrix, formula, or tibble.",
+    i = "not a {.obj_type_friendly {x}}."
```

Comment:
```suggestion
    i = "{.arg x} is a {.obj_type_friendly {x}}."
```

As written, I would expect the line above to end in a comma and for this line to be a continuation of the sentence (without additional punctuation.)